book/animations.md

```diff
@@ -1455,8 +1458,8 @@ class Browser:
             self.draw()
 ```
 
-Then, call `set_needs_raster` from the places that currently call
-`set_needs_raster_and_draw`, such as `handle_down`:
+Then, were we currently call `set_needs_raster_and_draw`, such as
```

Comment:
```suggestion
Then, where we currently call `set_needs_raster_and_draw`, such as
```

Reply:
I think "such as `handle_down`" maybe should be in a parenthetical, but if that's a copyeditor suggestion I'm willing to use commas.